Jenkinsfile

```diff
@@ -77,6 +77,7 @@ pipeline {
 
             // Run updatebot to update other repos
             sh './scripts/updatebot.sh'
+            sh 'jx step create version pr -n $GITHUB_ORG/$APP_NAME -v $(cat VERSION)'
```

Comment:
So what exactly does that do?

Reply:
I guess what I am asking here is whether one really wants to have each merge to master being merged directly into the version repo. Does that then really solve the version incompatibilities issues.

I guess if the jenkins-x-versions repo would indeed then run the BDD tests and verify that the upgraded version does not break anything, one would indeed get additional level security. 

At the same time, I am still not convinced that every merge to master should indeed do the release. Maybe I am stuck in some old type of thinking.

Reply:
What should happen here is that a PR with the new chart versions will get opened against https://github.com/jenkins-x/jenkins-x-versions. That PR should kick off a BDD test run to certify that the chart is working as expected before the release version is merged to the jenkins-x-versions branch if the BDD tests pass.

I'm currently working on the BDD test side to be able to run the BDD tests against apps.